integration/test/performancetest/performance_test.go

```diff
@@ -42,18 +64,44 @@ func TestPerformance(t *testing.T) {
 		t.Fatalf("Error: %v", err)
 	}
 
-	//------Placeholder to put data into database------//
-	//useless code so data get used and compiler isn't mad
 	if data == nil {
 		t.Fatalf("No data")
 	}
-	//data base 
+
+	//data base
 	dynamoDB := InitializeTransmitterAPI(DynamoDBDataBase) //add cwa version here
-	if dynamoDB == nil{
+	if dynamoDB == nil {
 		t.Fatalf("Error: generating dynamo table")
 	}
 	_, err = dynamoDB.SendItem(data)
-	if err !=nil{
+	if err != nil {
 		t.Fatalf("Error: couldnt upload metric data to table")
 	}
-}
\ No newline at end of file
+}
+
+func writeToLogs(t *testing.T, filePath string, durationMinutes time.Duration) {
+	f, err := os.Create(filePath)
+	if err != nil {
+		t.Fatalf("Error occurred creating log file for writing: %v", err)
+	}
+	defer f.Close()
+	defer os.Remove(filePath)
+
+	log.Printf("Writing lines to %s with %d transactions per second", filePath, transactionRatePerSecond)
+
+	startTime := time.Now()
+
+	//loop until the test duration is reached
+	for currTime := startTime; currTime.Sub(startTime) < durationMinutes; currTime = time.Now() {
+
+		//assume this for loop runs instantly for purposes of simple throughput calculation
+		for i := 0; i < transactionRatePerSecond; i++ {
```

Comment:
In anticipation of providing multiple different tps values, I think that this should be passed in as a function parameter